### ISSUE TITLE
Integrate typescript types

### ISSUE THREAD

**Issue by yannbriancon** [HarryStevens/geometric#43]
Currently, typescript types are given and maintained through DefinitelyTyped ([code](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/geometric), [npm](https://www.npmjs.com/package/@types/geometric)).

I saw some missing types like the new epsilons.
Would you consider adding the types to the repo and maintaining them here to avoid unsynchronised types?

**Comment by HarryStevens** [comment]
Yes, I really need to do this. I also got this suggestion in a [pull request](https://github.com/HarryStevens/geometric/pull/28).

I am not a TypeScript user myself. Can you recommend any good tutorials so I understand what I'd need to do to add this?
